inst/extdata/packages/flexsdm.yaml

```
@@ -0,0 +1,35 @@
+name : "flexsdm"
+title: "Tools for Data Preparation, Fitting, Prediction, Evaluation, and Post-Processing of Species Distribution Models"
+version: 1.3.3
+author: "Santiago J.E. Velazco, Brooke Rose, André F.A. Andrade, Ignacio Minoli, Janet Franklin"
+maintainer: "Santiago J.E. Velazco <sjevelazco@gmail.com>"
+cran: no
+github: https://github.com/sjevelazco/flexsdm
+description: "Assist in the construction of flexible
+      species distribution workflow by incorporating
+      comprehensive tools for data preparation, model
+      fitting, prediction, evaluation, and model
+      post-processing"
+occ_acquisition: no
+occ_cleaning: yes
+data_integration: no
+env_collinearity: yes
+env_process: no
+bias: yes
```

```suggestion
bias: no
```
Again, these functions seem more like occurrence cleaning than bias, but I'm not clear on the line @sylvainschmitt has conceptualized.